docs/developers/api.md

```diff
@@ -54,6 +54,19 @@ project_id -- the id of the project the sample is on
 
 list of sequencefile dictionary for given sample_id
 
+#### get_assemblies_files(self, project_id, sample_name)
+API call to api/projects/project_id/sample_id/assemblies
+We fetch the assemblies files through the project id on this route
+
+**arguments:**
+
+sample_name -- the sample id to get from irida, relative to a project
```

Comment:
If it is the sample id to get from irida shouldn't this read sample_id? If it is supposed to be sample_name add more to this line to make it clear

Reply:
Added some clarity here and in other places in the api that were a bit misleading,

It is actually the sample name that is used as the identifier, and not the sample id number.